docs/3.x/tech/installation-guide/cleanspeak-management-interface.adoc

```diff
@@ -49,9 +49,9 @@ Once the zip bundle has been extracted, the directory structure should look simi
 
 == Windows
 
-To install on Windows 7 or higher use the ZIP bundle. Extract the CleanSpeak Management Interface ZIP file anywhere on the file system. Remember where you extract the file. This location will be referred to as `CLEANSPEAK_HOME`. We suggest extracting this file to a directory such as `C:\inversoft` on Windows.
+To install on Windows 7 or higher, use the ZIP bundle. Extract the CleanSpeak Management Interface ZIP file anywhere on the file system. Remember where you extract the file. This location will be referred to as CLEANSPEAK_HOME. We suggest extracting this file to a directory such as C:\inversoft on Windows.
 
-Once the zip bundle has been extracted, the directory structure should look similar to this. If you installed somewhere other the default `CLEANSPEAK_HOME`, your directory structure will be different, this is only for shown as an example.
+Once the zip bundle has been extracted, the directory structure should look similar. If you installed somewhere other than the default CLEANSPEAK_HOME, your directory structure would be different; this is only shown as an example.
 
 [source,shell]
```

Comment:
We should update all of these examples to be use `cleanspeak` instead of `inversoft` for paths.